packages/core/src/modules/event/event-repository.ts

```diff
@@ -40,6 +41,22 @@ export class EventRepositoryImpl implements EventRepository {
     return events.map((e) => mapToEvent(e))
   }
 
+  async getAllByUserAttending(userId: string): Promise<Event[]> {
+    const event_ids_query = await this.db
```

Comment:
This should just do one query, not fetch ids, and then fetch Everything related to those id's

Reply:
implemented